src/spectrum-exporter.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @typedef {object} ExportOptions
+ * @property {string} columnDelimiter
+ * @property {string} stringDelimiter
+ * @property {boolean} quoteStrings
+ */
+
+/**
+ * @constructor
+ * @param {object} fftOutput
+ * @param {ExportOptions} [opts={}]
+ */
+export function SpectrumExporter(fftData, opts = {}) {
+  opts = _.merge(
+    {
+      columnDelimiter: ",",
+      quoteStrings: true,
+    },
+    opts,
+  );
+
+  /**
+   * @param {function} success is a callback triggered when export is done
+   */
+  function dump(success) {
+    const worker = new Worker("/js/webworkers/spectrum-export-worker.js");
+
+    worker.onmessage = (event) => {
+      success(event.data);
+      worker.terminate();
+    };
+
+    worker.postMessage({fftOutput: fftData.fftOutput,
+                        blackBoxRate: fftData.blackBoxRate,
+                        opts: opts});
+    }
+
+    // exposed functions
+    return {
+      dump: dump,
+    };
```

Comment:
Hmm, think I was wrong here. As I think you meant to return (39) in SpectrumExporter, not in worker.postMessage ?
Otherwise bracket on 36 should be on 42 :upside_down_face: 
```suggestion
      }

      // exposed functions
      return {
        dump: dump,
      };
```

Reply:
The normal text editor more comfortable, then github preview on smartphone :)